src/NuGet.Clients/NuGet.VisualStudio.Implementation/Extensibility/VsPackageInstaller.cs

```diff
@@ -320,7 +320,7 @@ private SourceRepository GetSource(string source)
 
                     foreach (var resource in metadataResources)
                     {
-                        tasks.Add(resource.GetVersions(dep.Id, token));
+                        tasks.Add(resource.GetVersions(dep.Id, Logging.NullLogger.Instance, token));
```

Comment:
In this case we should log to the output window, rather than the status bar. Is it reasonably possible to do it here or do we need a follow-up change?